### DISCUSSION TITLE
Conversion from ddf+Warp() to sitk.displacementFieldTransform + sitk.Resample()

### DISCUSSION THREAD

**Discussion by Lancial** [Project-MONAI/MONAI#3377] [Q&A]
Hi, I'm trying to evaluate my deformable registration result in SimpleITK but I found the following two operations don't produce the same visual result:

1. `Warp(source, flow)`
2.  first convert `flow` to a sitkVector Image, then to a `DisplacementFieldTransform`, finally do `sitk.Resample(source, flow)`

Specifically, even though both operations deform the original image in roughly the same direction, the second method deformed 'smaller', whereas the first method matches the target image better(the result of training).  

So my question is how do I obtain an accurate conversion from a predicted flow to a sitk.Transform? Or is my problem caused by some potential bug else where? Thank you so much.

**Comment by wyli** [comment]
If there's a more detailed test case I can look into it.
